app/mailers/application_mailer.rb

```diff
@@ -1,4 +1,11 @@
 class ApplicationMailer < ActionMailer::Base
   default from: "noreply@workdey.com"
   layout "mailer"
-end
+  default to: [
+                "olaide.ojewale@andela.com",
+                "chinedu.daniel@andela.com",
+                "amodu.temitope@andela.com",
+                "ruth.chukwumam@andela.com",
+                "workdeycomet@gmail.com"
+              ]
```

Comment:
Indent the right bracket the same as the start of the line where the left bracket is.